Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
```

Comment:
### [nokogiri](https://nokogiri.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.11.1 → 1.11.3

[_[change-log](https://nokogiri.org/CHANGELOG.html), [source-code](https://github.com/sparklemotion/nokogiri)_]





<details>
<summary>Commits</summary>

A change of **152** commits. See the full changes on [the compare page](https://github.com/sparklemotion/nokogiri/compare/v1.11.1...v1.11.3).

These are the first 10 commits:
- (8058d12) [doc: improve the help string for extconf.rb](https://github.com/sparklemotion/nokogiri/commit/8058d1222e32be69ccf7e147f7b974270719324e)
- (76017be) [test: expand testing for subclassing {XML,HTML}Document{Frag…](https://github.com/sparklemotion/nokogiri/commit/76017be56c89c95e504486991f34a259eae980c2)
- (e5646e8) [fix(jruby): call Document#initialize when creating via .pars…](https://github.com/sparklemotion/nokogiri/commit/e5646e813a5aeb2bbdbcebb807643ef84bf9af70)
- (8587972) [cleanup: rename internal method to nokogiri_xml_document_wra…](https://github.com/sparklemotion/nokogiri/commit/858797219259063bdca75e0330ef4f0726b7f40d)
- (c6dfeb9) [fix(cruby): ensure Document #initialize is called exactly on…](https://github.com/sparklemotion/nokogiri/commit/c6dfeb93bdd498ffb0a97369839eaf8073446b8a)
- (30c3a57) [update CHANGELOG.md](https://github.com/sparklemotion/nokogiri/commit/30c3a57a45973109f13a770d0dfbeb9b7ff46d99)
- (de426b4) [dev: rename default git branch to `main`](https://github.com/sparklemotion/nokogiri/commit/de426b47f1bb8e8f03b9156e04f6d0487f52c53e)
- (d79c2d0) [Merge pull request #2174 from sparklemotion/flavorjones-fix-…](https://github.com/sparklemotion/nokogiri/commit/d79c2d0ff44d34c82969e1453fe5da8667f97fa1)
- (fc436ad) [update LICENSE.md copyright line with date and additional au…](https://github.com/sparklemotion/nokogiri/commit/fc436ad6b53f3eb27e3d30ec874a499c466a8711)
- (51b3dbb) [dev: move definition of NOKOGIRI_SPEC from gemspec to Rakefi…](https://github.com/sparklemotion/nokogiri/commit/51b3dbb21d8a8841472510a115990b9e1375b187)


</details>